lib/BridgeXPathAbstract.php

```diff
@@ -0,0 +1,565 @@
+<?php
+
+abstract class BridgeXPathAbstract extends BridgeAbstract {
```

Comment:
Why not naming as XPathAbstract?

Reply:
I renamed it. Not sure though what naming choice supports clearity and understanding of its purpose mostly.